libs/core/langchain_core/prompts/chat.py

```diff
@@ -1338,11 +1338,11 @@ def _create_template_from_message_type(
     Raises:
         ValueError: If unexpected message type.
     """
-    if message_type in ("human", "user"):
+    if message_type in {"human", "user"}:
         message: BaseMessagePromptTemplate = HumanMessagePromptTemplate.from_template(
             template, template_format=template_format
         )
-    elif message_type in ("ai", "assistant"):
+    elif message_type in {"ai", "assistant"}:
```

Comment:
It probably doesn't matter much but do we know whether this is overall faster or slower? It requires both creating the data structure (unless it gets cached by the interpreter) and checking for membership.